### PR TITLE
Testing1

### PR DESCRIPTION
**Pull request by Kakalanp**
# Testing the project functions

### addTask refractor:
- Test-case 1: Testing if a task is added to an empty list
- Test-case 2: Testing if a task is added to an existing list

### deleteTask refractor:
- Test-case 1: Testing if a single item is removed from a list.
- Test-case 2: Testing if an array of items is removed from a list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ## Live Page
 
-[Page Link](https://kakalanp.github.io/TO-DO-list/)
+[Page Link](https://kakalanp.github.io/TO-DO-list/dist/)
 
 
 ## Getting Started
```
Comment (elisha2kyakpo1) [inline README.md:20]: - [x] Let's document how to go about testing the project(commands to run).

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi team,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- We should name our test files with the same name of the file we are testing, e.g `editTask` to be `editTask.test.js`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kakalanp) [reply README.md:20]: Added, thank you for the suggestion

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Kakalanp** , 

### **Congratulations! 🎉 Project Approved ✅**

Great work on making the changes requested by a previous reviewer :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**